Builds/CMake/deps/Findlibarchive_pc.cmake

```diff
@@ -20,3 +20,5 @@ find_path (LIBARCHIVE_INCLUDE_DIR
     ${libarchive_PC_INCLUDEDIR}
     ${libarchive_PC_INCLUDEDIRS}
   NO_DEFAULT_PATH)
+
```

Comment:
Some extraneous whitespace additions that you may want to exclude before the final patch is merged (also at the end of FindRocksDB.cmake above)

Reply:
cleaned those trailing blanks - thanks